builtins/amp-ad.js

```diff
@@ -257,6 +267,18 @@ export function installAd(win) {
       return loadPromise(this.iframe_);
     }
 
+    /** @override */
+    documentInactiveCallback() {
+      if (this.iframe_) {
+        this.paused_ = true;
+        // When the doc is inactive, hide the ads, so any work they do takes
+        // less CPU and power.
+        this.iframe_.style.display = 'none';
```

Comment:
`Display:none`. Visibility has all of the cost with the downside of the ad not showing.